src/components/pages/divisionPage/divisionPage.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import {
+  Box,
+  Heading,
+  HorizontalRule,
+  Link,
+  SimpleGrid,
+} from "@nypl/design-system-react-components";
+import PageLayout from "../../pageLayout/pageLayout";
+import { useNumColumns } from "src/hooks/useNumColumns";
+import { useParams } from "next/navigation";
+import { headerBreakpoints } from "src/utils/breakpoints";
+import { slugToString } from "src/utils/utils";
+import CollectionCard from "src/components/cards/collectionCard";
+import { CollectionCardModel } from "src/models/collectionCard";
+import useBreakpoints from "src/hooks/useBreakpoints";
+import CollectionDataType from "src/types/CollectionDataType";
+import { mockCollections } from "__tests__/__mocks__/data/mockCollections";
+
+export default function DivisionPage() {
+  const params = useParams();
+  const { isLargerThanLargeTablet } = useBreakpoints();
+  const slug = params.slug as string;
+  const title = slugToString(slug);
```

Comment:
I think eventually this might need to come from the API response. For example, https://digitalcollections.nypl.org/divisions/irma-and-paul-milstein-division-of-united-states-history-local-history

The slug does not match the title, there is a missing comma and a word. Not a big deal _right now_ but keep this in mind. CC @avertrees